src/main/java/it/gov/pagopa/payhub/activities/dto/export/debtposition/PaidInstallmentExportFlowFileDTO.java

```diff
@@ -0,0 +1,187 @@
+package it.gov.pagopa.payhub.activities.dto.export.debtposition;
+
+import com.opencsv.bean.CsvBindByName;
+import com.opencsv.bean.CsvIgnore;
+import it.gov.pagopa.payhub.activities.enums.EntityIdentifierType;
+import it.gov.pagopa.payhub.activities.enums.UniqueIdentifierType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class PaidInstallmentExportFlowFileDTO {
+    private static final String EXPORT_PAID_VERSION_V1= "v1.0";
+    private static final String EXPORT_PAID_VERSION_V1_1= "v1.1";
+    private static final String EXPORT_PAID_VERSION_V1_2= "v1.2";
+    private static final String EXPORT_PAID_VERSION_V1_3= "v1.3";
```

Comment:
Let's create a "ExportConstants" class and move these there.
When we will introduce the exportTrackingVersion (or whetever will be its name) on the ExportFile entity, we will validate its value based on the requested ExportFileType, thus probably we will next introduce a Map<ExportFileType, Set<String>> to collect the available versions based on the file type requested

Reply:
ps: we could already introduce the Map fill it with Paid requirements